data.sql

```diff
@@ -136,4 +136,75 @@ INSERT INTO animals(
     3,
     TRUE,
     17
-);
\ No newline at end of file
+);
+
+--Sam Smith 34 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34);
+
+--Jennifer Orwell 19 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Jennifer Orwell', 19);
+
+--Bob 45 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Bob', 45);
+
+--Melody Pond 77 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Melody Pond', 77);
+
+--Dean Winchester 14 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Dean Winchester', 14);
+
+--Jodie Whittaker 38 years old.
+INSERT INTO owners (full_name, age)
+VALUES ('Jodie Whittaker ', 38);
+
+-- Pokemon
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+-- Digimon
+INSERT INTO species (name)
+VALUES ('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value:
+-- If the name ends in "mon" it will be Digimon
+UPDATE animals 
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name LIKE '%mon';
+
+-- All other animals are Pokemon
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Pokemon')
+WHERE species_id IS NULL;
+
+-- Modify your inserted animals to include owner information (owner_id):
+-- Sam Smith owns Agumon.
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith')
+WHERE name = 'Agumon';
+
+-- Jennifer Orwell owns Gabumon and Pikachu.
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell')
+WHERE name IN ('Gabumon', 'Pikachu');
+
+
+-- Bob owns Devimon and Plantmon.
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob')
+WHERE name IN ('Devimon', 'Plantmon');
+
+-- Melody Pond owns Charmander, Squirtle, and Blossom.
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond')
+WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+
+
+-- Dean Winchester owns Angemon and Boarmon.
+UPDATE animals 
+SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
+WHERE name IN ('Angemon', 'Boarmon');
```

Comment:
- [ ] <b>REQUIRED:</b> These queries should be moved to `queries.sql`, kindly do so to comply with project requirements.